indexer/src/main/kotlin/com/openlattice/indexing/BackgroundIndexedEntitiesDeletionService.kt

```diff
@@ -92,47 +98,65 @@ class BackgroundIndexedEntitiesDeletionService(
             val w = Stopwatch.createStarted()
             //We shuffle entity sets to make sure we have a chance to work share and index everything
             val lockedEntitySets = entitySets.values
+                    .map { EntitySetForDeletion(it.id, it.name, it.partitions) }
```

Comment:
This was changed very recently ... just flagging this to double check with @makosblade

Reply:
yeah I didn't change anything about the recent pattern, EntitySetForDeletion is effectively the same as IndexerEntitySetMetadata minus entityTypeId to accommodate deleted entity sets.